extras/tasks/iconFont/index.js

```diff
@@ -7,16 +7,17 @@ var generateIconSass = require('./generateIconSass')
 var handleErrors     = require('../../lib/handleErrors')
 var package          = require('../../../package.json')
 var path             = require('path')
+var projectPath      = require('../../lib/projectPath')
```

Comment:
This path seems wrong regarding the actual file structure of the problem as it lives in `extras/tasks/iconFont` which means it's outside of `gulpfile.js` folder. So to reference `projectPath.js` file, it should be something like `../../../gulpfile.js/lib/projectPath`.

It's the only first I didn't know on to test, so I decided to follow the  ` var handleErrors     = require('../../lib/handleErrors')` pattern above.

Either I'm wrong or `handleErrors` is currently pointing at the wrong directory.

Reply:
Oups ping @benjtinsley

Reply:
hmm looks like you discovered a bug here. unless there is something i am totally misinterpreting, there is no way  `handleErrors` can find the correct path without including the `gulpfile.js/` directory in the path. 

i think your reasoning is correct. could you update the `projectPath` path to your initial offering (`../../../gulpfile.js/lib/projectPath`) as well as updating the `handleErrors` path? i think the `iconFont` task may not get used by very many people so this was never discovered until now 😉